hugo/config/_default/menus.toml

```diff
@@ -45,3 +45,15 @@
     weight = 2
     identifier = "privacy"
     url = "/privacy"
+
+[[footer]]
+    title = "Contact"
+    weight = 3
+    identifier = "contact"
+    url = "/contact"
+
+[[footer]]
+    title = "Sponsors"
+    weight = 4
+    identifier = "sponsors"
+    url = "/sponsors"

```

Comment:
Correct position?

Reply:
Are you referring to the entry's position in the footer? I don't mind where that is listed :shrug:

Reply:
Yes. Ok, I guess we will see what others have to say on review.